app/assets/javascripts/discourse/app/pre-initializers/discourse-bootstrap.js

```diff
@@ -19,14 +19,12 @@ export default {
     if (isTesting()) {
       return;
     }
-    const preloadedDataElement = document.getElementById("data-preloaded");
+    const preloadedData = window.preloaded;
```

Comment:
Premature feedback, perhaps, but we should use a much more complex variable name here to avoid conflicts since it's a global namespace.